jobs/payment-jobs/tasks/cfs_create_account_task.py

```diff
@@ -111,7 +111,9 @@ def create_accounts(cls):  # pylint: disable=too-many-locals
 
             except Exception as e:  # NOQA # pylint: disable=broad-except
                 # publish to mailer queue.
-                mailer.publish_mailer_events('PadSetupFailed', pay_account)
+                if pay_account.payment_method == PaymentMethod.PAD.value:
```

Comment:
This is a fix to not send failure emails for Online Banking accounts, as these accounts doesnt have any pay information.